Formula/codeclimate.rb

```diff
@@ -1,19 +1,18 @@
-require "formula"
-
 class Codeclimate < Formula
-  CODECLIMATE_VERSION = "0.24.2".freeze
-
+  desc "Code Climate CLI"
   homepage "https://github.com/codeclimate/codeclimate"
-  version CODECLIMATE_VERSION
-
-  url "https://github.com/codeclimate/codeclimate/archive/v#{CODECLIMATE_VERSION}.tar.gz"
-  sha1 "c28ce24aea878ecd8b1bd65cc313206bcd36aece"
+  url "https://github.com/codeclimate/codeclimate/archive/v0.24.2.tar.gz"
+  sha256 "f6ab47d602c8a6ec2d4e9f86df829cc31c3059ad025058c7646499b154242351"
 
   def install
     # Alter PATH to ensure `docker' is available
-    system "env", \
-      "PATH=#{HOMEBREW_PREFIX}/bin:#{ENV["PATH"]}", \
-      "PREFIX=#{prefix}", "make", "install"
+    if Formula["docker"].linked_keg.exist?
```

Comment:
That seems reasonable, the solution for this user is almost certainly to `--force` it.

I have a few questions though (for the general audience):

1. How did this happen?

  How is there a `/usr/local/bin/docker` present, _and_ an (unlinked) `docker` keg? How might one end up in this scenario (understanding it's probably PEBCAK)?

1. How do we support users that _don't_ have a brew-installed `docker`?

  There are many reasons a user might want to do this, and IMO we should support them.

  I'm happy to be wrong, but I'm 99% sure that the `$PATH` of the calling shell can't impact the `$PATH` seen by the Formula when it runs (otherwise we wouldn't be having this conversation). So a user can't do the simple thing of:

   ```
   PATH=/my/docker/bin:$PATH brew install codeclimate
   ```

  My vote would be to prepend some reasonable locations that, for whatever reason, Homebrew decided shouldn't be in the in-Formala `PATH`, of which `/usr/local/bin` should be one.

/cc @GordonDiggs